src/block/Moss.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+/*
+ *
+ *  ____            _        _   __  __ _                  __  __ ____
+ * |  _ \ ___   ___| | _____| |_|  \/  (_)_ __   ___      |  \/  |  _ \
+ * | |_) / _ \ / __| |/ / _ \ __| |\/| | | '_ \ / _ \_____| |\/| | |_) |
+ * |  __/ (_) | (__|   <  __/ |_| |  | | | | | |  __/_____| |  | |  __/
+ * |_|   \___/ \___|_|\_\___|\__|_|  |_|_|_| |_|\___|     |_|  |_|_|
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * @author PocketMine Team
+ * @link http://www.pocketmine.net/
+ *
+ *
+ */
+
+declare(strict_types=1);
+
+namespace pocketmine\block;
+
+use pocketmine\item\Item;
+
+class Moss extends Opaque{
```

Comment:
This class isn't needed. You can use Opaque instead.

Reply:
I find that for me having the blocks in a file allows the developer to know which block is made and others don't like me who wants to implement all the blocks that pocketmine does not support

Reply:
yes it's not wrong but it would be a way that might be useless for you to know which block is already implemented in pocketmine but hey for that we have the VanillaItems just to do a Ctrl + F to search for the block

Reply:
The block will be added back anyways once the spread feature gets implemented.

Reply:
OK 👌